### ISSUE TITLE
SQLLiteral issue

### ISSUE THREAD

**Issue by zdnk** [groue/GRDB.swift#566]
### What did you do?

```
protocol Dirtifiable {
    
    static func dirtify(_ db: Database) throws
    static func cleanup(_ db: Database) throws
    
}

extension Dirtifiable where Self: PersistableRecord {
    
    static func dirtify(_ db: Database) throws {
        try db.execute(literal: "UPDATE \(self) SET dirty = 1")
    }
    
    static func cleanup(_ db: Database) throws {
        try db.execute(literal: "DELETE FROM \(self) WHERE dirty = 1")
    }
    
}
```

### What did you expect to happen?

Build.

### What happened instead?

> Cannot convert value of type 'String' to expected argument type 'SQLLiteral'

### Environment

**GRDB flavor(s):** GRDB
**GRDB version:** 4.1.1
**Installation method:** (manual)
**Xcode version:** 10.2.1
**Swift version:** 5
**Platform(s) running GRDB:** iOS

**Comment by groue** [comment]
Hello @zdnk.

Please explain why you think your code should build. This may help you spot where your app, or GRDB, may need to be modified.

**Comment by zdnk** [comment]
Well, I believe this worked fine before with static members (`\(self)`). When I moved the code to another project with updated GRDB version, it stopped working. I believe that you can use Swift 5 string interpolation with `SQLLiteral` and `TableRecord`.

**Comment by groue** [comment]
Did you check that your project is compiled in Swift 5 m?

**Comment by zdnk** [comment]
Yes, it is.
<img width="497" alt="Screenshot 2019-07-11 at 18 03 34" src="https://user-images.githubusercontent.com/2551911/61066660-3fe14e00-a406-11e9-9b6b-61c0ee6450b7.png">

**Comment by zdnk** [comment]
I figured it out.
I am using Carthage to build the GRDB using following script to work around the issues with schemes GRDB has with Carthage:
```
#!/bin/sh

echo "Checking out dependencies"
carthage checkout

echo "Removing GRDBCustom.xcodeproj" 
rm -r Carthage/Checkouts/GRDB.swift/GRDBCustom.xcodeproj

echo "Building invalid dependencies"
carthage build --platform ios --cache-builds --no-use-binaries
```

However the `GRDB.xcodeproj` has Swift version set to 4.2. 
I will make PR to fix this and set it to 5.0, if it is ok with you?

**Comment by groue** [comment]
Superseded by #568
